core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4671,6 +4671,13 @@ public String toString() {
               + "an exponential backoff. This property determines the maximum duration to retry for"
               + " before giving up. Note that, this value is set to 5s for fs and fsadmin CLIs.")
           .build();
+  public static final PropertyKey USER_RPC_CALL_DEADLINE_DURATION =
+      new Builder(Name.USER_RPC_CALL_DEADLINE_DURATION)
+          .setDefaultValue("30sec")
```

Comment:
@ggezer I have two questions:

1. I'm pretty sure this isn't a valid default value? It is a string with the suffix `"sec"` but [gRPC stub's withDeadlineAfter](https://grpc.github.io/grpc-java/javadoc/io/grpc/stub/AbstractStub.html#withDeadlineAfter-long-java.util.concurrent.TimeUnit-) expects a `long`. Also we use `TimeUnit.MILLISECONDS` so shouldn't this be `30000`?

2. Should we let users configure the deadline duration `TimeUnit` as a property too? Otherwise we should mention in the description below that it is always `TimeUnit.MILLISECONDS`.

Reply:
I provided places where your questions are answered. 
Having said that, I'll anyway provide a follow up PR as I discovered not all sync stubs are covered in this PR